server/controllers/konnectors.coffee

```diff
@@ -23,10 +23,17 @@ module.exports =
                     konnector.injectEncryptedFields()
 
                 # Add customView field
-                    konnectorModule = require "../konnectors/#{konnector.slug}"
-
-                    if konnectorModule.customView?
-                        konnector.customView = konnectorModule.customView
+                konnectorModule = require(
+                    path.join(
+                        "../konnectors/",
```

Comment:
If we rely on `path.join`, then a proper way to write it must be `path.join '..', 'konnectors', konnector.slug`, nope?